### DISCUSSION TITLE
Document with multiple pages

### DISCUSSION THREAD

**Discussion by hfgd123** [community/community#83435] [GitHub Education]
### Hi everyone,

Hi. 
Why can I only upload one picture of my student ID? My Name/Adress/DOB is on one side, the date until when it is valid is on the other side, so I can't get approved. Is there any way for me to get student benifits?
Thank you

**Comment by faridnec** [comment]
Hi, I also had my card without the date when I applied Student pack but they accepted it. Just make sure the photo is clear and shows proof of an official school you attend to. Give it a go!
